.editorconfig

```diff
@@ -0,0 +1,36 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Inconsistent with the PSR-2 coding standard. This should be changed or there should be a section for php.